Zend/zend_API.c

```diff
@@ -331,13 +331,13 @@ ZEND_API int ZEND_FASTCALL zend_parse_arg_class(zval *arg, zend_class_entry **pc
 	*pce = zend_lookup_class(Z_STR_P(arg));
 	if (ce_base) {
 		if ((!*pce || !instanceof_function(*pce, ce_base))) {
-			zend_argument_type_error(num, "must be a class name derived from %s, '%s' given", ZSTR_VAL(ce_base->name), Z_STRVAL_P(arg));
+			zend_argument_type_error(num, "must be a class name derived from %s, \"%s\" given", ZSTR_VAL(ce_base->name), Z_STRVAL_P(arg));
```

Comment:
The conjunction of quoted and unquoted class name here seems a bit odd. As we have a valid class here (which will use reasonable characters), I'd drop the quotes.

Reply:
Yeah, makes sense! :)